test/functions_test.rb

```diff
@@ -162,10 +195,49 @@ def returns_a_color
         return Script::Color.new(red: 0, green: 0, blue: 0)
       end
 
+      def returns_a_number
+        return Sass::Script::Value::Number.new(-312,'rem')
+      end
+
+      def inspect_number ( argument )
+        raise StandardError.new "passed value is not a Sass::Script::Value::Number" unless argument.is_a? Sass::Script::Value::Number
+        return argument
+      end
+
+      def inspect_map ( argument )
+        argument.to_h.each_pair do |key, value|
+          raise StandardError.new "key #{key.inspect} is not a string" unless key.is_a? Sass::Script::Value::String
+
+          valueClass = case key.value
+                         when 'string'
+                           Sass::Script::Value::String
+                         when 'number'
+                           Sass::Script::Value::Number
+                         when 'color'
+                           Sass::Script::Value::Color
+                         when 'map'
+                           Sass::Script::Value::Map
+                       end
+
+          raise StandardError.new "unknown key #{key.inspect}" unless valueClass
+          raise StandardError.new "value for #{key.inspect} is not a #{valueClass}" unless value.is_a? valueClass
+        end
+        return argument
+      end
+
```

Comment:
> Rather than just returning the map, it would be nice to test that the map values are usable within the ruby function. So perhaps you could pass in a map with a string, number, etc, and ensure that within the ruby function they are usable and have the correct types.

The above 2 methods (`inspect_number` & `inspect_number`) inspect values passed to them. If you know of a way to use asserts instead of raising errors in here then please do tell. Same if you know of a better/more specific error to use, or if you have a better strategy to make tests fail from within these 2 methods.